frontend/src/app/modules/bookings/bookings-page/bookings-page.component.ts

```diff
@@ -94,17 +149,21 @@ export class BookingsPageComponent extends BaseComponent implements OnInit {
         if (width < tabletMaxWidth) {
             this.numberOfMembersToDisplay = 1;
         }
-        if (width < phoneMaxWidth) {
-            this.numberOfMembersToDisplay = 1;
-        }
         if (width < widthToContainZeroItemUpperLimit) {
             this.numberOfMembersToDisplay = 0;
         }
+        if (width < phoneMaxWidth) {
+            this.numberOfMembersToDisplay = 1;
+        }
     }
 
     private getPageSize(): number {
         const bookingContainer = this.el.nativeElement.getElementsByClassName('container')[0] as HTMLElement;
 
         return bookingContainer.offsetWidth;
     }
+
+    private reloadBookings() {
```

Comment:
Why do we need this extra method with one call to another method&

Reply:
Fixed it.